docs/content/guides/recipes/terraform/howto-custom-provider/snippets/env-complete.bicep

```diff
@@ -55,7 +57,13 @@ resource env 'Applications.Core/environments@2023-10-01-preview' = {
         }
       }
       env: {
-        PGHOST: 'postgres.corerp-resources-terraform-pg-app.svc.cluster.local'
+        PGPORT: '5432'
+      }
+      envSecrets: {
+        PGHOST: {
+          source: pgsSecretStore.id
+          key: 'host'
```

Comment:
I think you also need to add the `{ host: myhost }` entry into the `pgsSecretStore`?

Reply:
ln 24-26

Reply:
sorry I missed this! pls disregard and i'll resolve this comment